src/components/Header/Header.component.jsx

```diff
@@ -27,9 +30,16 @@ const Center = styled(Col)`
   align-items: center;
 `;
 
-const Header = ({ onSearch, onToggle }) => {
+const Header = ({ onToggle }) => {
+  const { dispatch } = useContext(Context);
+
+  const searchVideosAction = async (searchTerm) => {
+    const res = await searchVideos(searchTerm);
+    dispatch({ type: 'SEARCH_VIDEOS', payload: res.result.items });
```

Comment:
In the test coverage this lines are marked as untested, you can fire an event to trigger the `handleSearch` which will trigger this code